### ISSUE TITLE
Projects API returns relative links to images

### ISSUE THREAD

**Issue by alanmoo** [mozilla/science.mozilla.org#280]
Some events like https://api-mozillascience-staging.herokuapp.com/events/3/ return relative links to images stored inside the project repo itself. Should we change this?

@gvn @alicoding @gideonthomas @acabunoc

**Comment by acabunoc** [comment]
I think we can! In this case, it's a recurring event we create via template -- all the template images you'll find will be relative links stored inside the repo.

**Comment by gvn** [comment]
Are events administered through Django? I ideally they should probably live on a CDN.

**Comment by alicoding** [comment]
@gvn can you elaborate on that?

**Comment by gvn** [comment]
Sure! So, if images are being added through Django's web interface then we should add some kind of CDN publishing to also occur when changes are published. This service may be used by other clients as well and that creates an implicit extra dependency on the Science site as a image host.

TLDR: It's ideal that we get back a fully qualified URL for images in the payload coming from the service.

**Comment by acabunoc** [comment]
@gvn I agree. Most of the images in events are stored in the repo because it was easier at the time. 

Long term (phase 2) events will be created via web interface (not Django's web interface).

In the current [create event page](https://www.mozillascience.org/community/new), you upload the image to s3 (unless you're making a recurring event). So some of our newer events have a full URL.

**Comment by gvn** [comment]
Ok, cool. So it sounds like we should just go through and upload anything with a relative path to S3. We'll see some broken images on staging, which should help surface that work.

**Comment by acabunoc** [comment]
Just a note: you'll find the same problem on our projects! however, we switched to S3 much earlier there, so more of them have the full path.

**Comment by alanmoo** [comment]
I don't think I have credentials for S3, who can help with that?

**Comment by acabunoc** [comment]
I will update the data currently in Django & upload missing images to s3!

**Comment by gideonthomas** [comment]
@alanmoo is this good to close now?

**Comment by acabunoc** [comment]
I only did events - did  you do all the projects? If you're done projects, then yes!

**Comment by alanmoo** [comment]
@acabunoc Event 9 is still broken

**Comment by acabunoc** [comment]
missed that! should be good now

**Comment by gideonthomas** [comment]
I did all projects, @acabunoc feel free to close this
